rpcMessages/rpcMessages.cpp

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
Commit message seems to have been truncated.
Also, it is best to write 'change' instead of 'Changing', 'increase' instead of 'increasing'.

Reply:
:)